compute/disks/disks_test.go

```diff
@@ -465,4 +465,40 @@ func TestComputeDisksSnippets(t *testing.T) {
 		// Cannot clean up the disk just yet because it must be done after the VM is terminated.
 		// It will be done by deleteInstance function.
 	})
+	t.Run("Create a test disk and list it", func(t *testing.T) {
+		var buf bytes.Buffer
+
+		if err := listDisks(&buf, tc.ProjectID, zone, ""); err != nil {
+			t.Fatalf("listDisks got err: %v", err)
```

Comment:
```suggestion
			t.Errorf("listDisks got err: %v", err)
```
while it is true this test fails on an error here, we can still continue. so this should be `Errorf` instead. The same is true for the two Fatalfs below.

Reply:
fixed